warrant/aws_srp.py

```diff
@@ -165,3 +166,19 @@ def process_challenge(self, challenge_parameters):
                 "USERNAME": user_id_for_srp,
                 "PASSWORD_CLAIM_SECRET_BLOCK": secret_block_b64,
                 "PASSWORD_CLAIM_SIGNATURE": signature_string.decode('utf-8')}
+
+    def authenticate_user(self):
+        self.client = boto3.client('cognito-idp')
```

Comment:
Ok I have a solution. Add client=None as a kwarg in __init__ and add self.client = client. Also add client=None as a kwarg in the authenticate_user method. Inside the method add an if statement that either uses self.client or the local client kwarg.